blog/_posts/2023-05-26-scala-3.3.0-released.md

```diff
@@ -0,0 +1,162 @@
+
+The LTS model doesn't change anything in our compatibility guarantees. Projects built with all future releases will be able to depend on any library compiled with Scala 3.3.x. This is just our standard guarantee that the newer compiler can always consume the output of the older version.
+
+LTS might be a great choice for library authors who can now receive constant bug fixes and developer experience improvements without forcing the users of the library to update the compiler version in their project.
+
+You can read more about our compatibility guarantees in [the recent blog post](https://virtuslab.com/blog/the-scala-3-compatibility-story/).
+
+## What's new in Scala 3.3.0
+
+### Linting
+
+Scala 3.3.0 brings back the linting to the Scala compiler.  Right now, you can enable checking for unused symbols and discarded values. More linting options will come soon in the following Scala 3.3.x releases.
+
+#### Checking for unused values
+
+There is an entire family of compiler options added in 3.3.0 for checking and reporting different kinds of unused symbols:
+
+- `-Wunused:imports` - for unused imports
+- `-Wunused:privates` - for unused local definitions
+- `-Wunused:locals`  - for unused local definitions
+- `-Wunused:explicits` - for unused explicit parameters
+- `-Wunused:implicits` - for unused implicit parameters (parameters in `using` clauses)
+- `-Wunused:params` - for all unused method parameters
+- `-Wunused:all` - for enabling all of the above lints
+
+#### Checking for discarded values
+
+Discarding non-unit values is usually the symptom of subtle mistakes. The compiler can now warn you about all discarded values saving you from bugs resulting from those hard-to-spot mistakes.
+
+In the following simplified example
+
+```Scala
+def failure: Either[String, Nothing] = Left("something broke")
+
+def failedComputation: Either[String, Unit] =
+  Right(()).map(_ => failure)
+```
+
+the programmer, by mistake, used the `map` method instead of `flatMap`. The code still compiles, but due to value discarding, behaves unexpectedly, returning `Right(())` from the `failedComputation` method. If the `-Wvalue-discard` flag is enabled, the compiler will report the warning, saving the user from a potential bug.
+
+### More consistent brace-less syntax
+
+Braces around method parameters can now be replaced with a colon. This can lead to cleaner, shorter, and often more readable code in places like configuration DSLs or test case definitions.
+
+```Scala
+def canFail(input: String) = Try:
+  someComputation(input).flatMap: res =>
+    val partial = moreComputation(res)
+    andEvenMore(partial)
+
+class TestSuite extends munit.FunSuite:
+  test("the job doesn't fail"):
+    val job = canFail("some data")
+    assert(job.isSuccess)
+```
+
+### `boundary` and `break`
+
+Two new methods were added to the standard library in the place: `boundary` and `brake`. They are safer and more expressive replacements for recently deprecated non-local returns.
```

Comment:
```suggestion
Two new methods were added to the standard library in the place: `boundary` and `break`. They are safer and more expressive replacements for recently deprecated non-local returns.
```

Reply:
It seems something is missing after “in the place”, did you mean “in place of `scala.util.control.Breaks` and `scala.util.control.NonLocalReturns`”?